v2/mongodb-to-googlecloud/docs/MongoDbToBigQuery/README.md

```diff
@@ -141,3 +153,31 @@ gcloud beta dataflow flex-template run ${JOB_NAME} \
         --template-file-gcs-location=${TEMPLATE_IMAGE_SPEC} \
         --parameters mongoDbUri=${MONGODB_HOSTNAME},database=${MONGODB_DATABASE_NAME},collection=${MONGODB_COLLECTION_NAME},outputTableSpec=${OUTPUT_TABLE_SPEC},userOption=${USER_OPTION}
 ```
+
+### UDF:
```

Comment:
Can this section be named "Text Transformer UDFs" similar to https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/v2/pubsub-cdc-to-bigquery
and include a similar description (omitting the parts about python).

Reply:
Updated as per suggestion